File: Tasty Decisions.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int x=sc.nextInt();
            int y=sc.nextInt();
            if(2*x>5*y)
            System.out.println("Chocolate");
            else if(2*x<5*y)
            System.out.println("Candy");
            else
            System.out.println("Either");
        }
    }
}

File: Tasty Decisions.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int a,b,cho,ca;
        scanf("%d%d",&a,&b);
        cho=2*a;
        ca=5*b;
        if(cho>ca)
        printf("Chocolate\n");
        else if(cho<ca)
        printf("Candy\n");
        else
        printf("Either\n");
    }
}
